templates/zerver/portico.html

```diff
@@ -67,6 +67,12 @@
           <a href="{{ url('register') }}">{{ _('Register') }}</a>
           {% endif %}
       </li>
+      <li><span class="little-bullet">·</span></li>
+      <li>
+          {% if open_realm_creation %}
+          <a href="/create_realm">{{ _('Create new realm') }}</a>
```

Comment:
Probably should call it "Create new organization" for consistency with how we refer to realms in documentation